agent/wsclient/client_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const dockerEndpoint = "/var/run/docker.sock"
```

Comment:
Please ignore this comment.  I read this too quickly and did not realize it was just part of the unit test.